src/Header.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+const Header = (props) => {
+    return (
+        <header className="header">
+            <h1>todos</h1>
+
+            <input
+                className="new-todo"
+                placeholder="What needs to be done?"
+                autoFocus=""
+                onKeyDown={(event) => {
```

Comment:
Wrap input with `form` and you might not need to subscribe to `onKeyPress` event

Reply:
@ivadimko I am not sure that form needed for one element, cause it will add more code, can you pls tell main points doing so, except for no need is subscribing.